api/module.mk

```diff
@@ -36,11 +36,19 @@ $(BUILD): $(SCM_SRC) $(SCM_RESOURCES)
 	@$(TOUCH) $@
 
 LOGFILE := $(DERIVED_MODULE_DIR)/test.log
-$(CHECK): $(SCM_SRC) $(SCM_RESOURCES) $(CLOJURE_PROJECT)
+$(UNIT): $(SCM_SRC) $(SCM_RESOURCES) $(CLOJURE_PROJECT)
```

Comment:
You need to rename this in `modules.mk`, both the timestamp and the phony target

Reply:
Thanks. Why do we create result files for phony targets?

Reply:
`make` will always try to make `phony` targets. By using a time stamp, we only need to make each target when its dependencies are newer than the time stamp.